ansible/roles/passenger/tasks/main.yml

```diff
@@ -48,7 +48,7 @@
       dest: /etc/apache2/ssl/intermediate.crt
     sudo: true
 
-  - name: Create Rails Apache config (SSL Enabled)
```

Comment:
This isn't the apache config it is the rails config for apache (e.g. it configures passenger, railsEnv, etc).